### PR TITLE
[Easy] Token type cleanup

### PR DESCRIPTION
**Pull request by nlordell**
Just cleanup some of the token types that were recently introduced.

### CODE REVIEW

**Review by e00E — APPROVED** [review]

**Review by josojo — APPROVED** [review]
Clean of the day!

File: driver/src/price_estimation/mod.rs [hunk 1]
```diff
@@ -68,10 +68,10 @@ mod tests {
...
     fn token_get_price() {
         for (token, usd_price, expected) in &[
-            (Token::test(4, "USDC", 6), 0.99, 0.99 * 10f64.powi(30)),
-            (Token::test(7, "DAI", 18), 1.01, 1.01 * 10f64.powi(18)),
-            (Token::test(42, "FAKE", 32), 1.0, 10f64.powi(4)),
-            (Token::test(99, "SCAM", 42), 10f64.powi(10), 10f64.powi(4)),
+            (Token::new(4, "USDC", 6), 0.99, 0.99 * 10f64.powi(30)),
+            (Token::new(7, "DAI", 18), 1.01, 1.01 * 10f64.powi(18)),
+            (Token::new(42, "FAKE", 32), 1.0, 10f64.powi(4)),
+            (Token::new(99, "SCAM", 42), 10f64.powi(10), 10f64.powi(4)),
```
Comment (bh2smith) [inline driver/src/price_estimation/mod.rs]: I see that we label these tokens as fake and scam, but does the inflated price have anything to do with it or are we just demonstrating that we can have very large prices?

**Review by bh2smith — COMMENTED** [review]
Squeaky clean.

File: driver/src/price_estimation/mod.rs (hunk 1 above)
Comment (nlordell) [reply driver/src/price_estimation/mod.rs]: It is to test tokens with more than 18 and more than 36 decimals respectively. More than 36 decimals is a special case as you would expect the OWL price the be 0 even when the USD price is `1.0`, this just tests that if the token has more than 36 decimals it still works for very large USD prices.

Comment (bh2smith) [reply driver/src/price_estimation/mod.rs]: Oh wow. I didn't even think about ever having more than 18 decimals!

**Pull request merged**